src/menupage.js

```diff
@@ -1,8 +1,8 @@
-const Menu = (() => {
-  
-  const loadMenuPage = (arg) => {
-  	contentDiv.innerHTML = menuContent;
-  }
-
+/* eslint-disable no-mixed-spaces-and-tabs,
+import/prefer-default-export, no-param-reassign, no-tabs */
```

Comment:
Please, let's export the module using default export instead of ignoring the rule.

adding the following, in this case, should solve the linter warning instead of ignoring it.
```js
export default Menu
```

Also, `npx eslint src --fix` should fixed the `no-mixed-spaces-and-tabs` rule.